### ISSUE TITLE
Greater visual distinction for code blocks in site theme?

### ISSUE THREAD

**Issue by cmgrote** [odpi/egeria#711]
This may purely be down to my own personal preferences (so happy for others to chime in), but I'm finding it somewhat challenging to distinguish the code blocks in our documentation.  For example:

<img width="969" alt="screenshot 2019-02-04 at 16 37 21" src="https://user-images.githubusercontent.com/5864283/52222451-6d482f80-289b-11e9-8d8b-d84f984cfa39.png">

We have a swap to a mono-spaced font, which is good, but I much prefer having a boxed background / outline around these as well, with padding, so that they stand out from the rest of the content, eg:

<img width="919" alt="screenshot 2019-02-04 at 16 42 41" src="https://user-images.githubusercontent.com/5864283/52222662-f19ab280-289b-11e9-94a4-b391b53586da.png">

For me this helps visually break-up the page into what appear to be a number of super-simple instructions and examples; without it I have the visual feeling that the instructions are a complicated.  (It also makes it easier for me to distinguish what I might want to copy / paste 😉)

Also, I see that these code blocks themselves are still being truncated when they're wider than the page, eg:

<img width="1010" alt="screenshot 2019-02-04 at 16 46 08" src="https://user-images.githubusercontent.com/5864283/52222852-708feb00-289c-11e9-9950-f44dd462e340.png">

(Rather than being able to scroll these "boxes" horizontally as you are in most themes -- not sure if that's related or not to the background boxing (?))

**Comment by cmgrote** [comment]
@alexcontini ^^ we discussed the above on a call this morning (amongst other topics), and there was broad support for making this kind of change.  Is this something you could investigate?

**Comment by alexcontini** [comment]
@cmgrote  ah yes! I will fix this asap. Working on a solution and will open a PR soon

**Comment by alexcontini** [comment]
@cmgrote I realized before I should create a PR for this I should run the design by you. I thought these three design options might work? One with a light grey outline, one with a black outline and one with a black background. Let me know any thoughts when you have a chance!

![screen shot 2019-02-11 at 4 27 55 pm](https://user-images.githubusercontent.com/27962005/52573275-1225b800-2e1a-11e9-97f6-8f750f3cdd14.png)
![screen shot 2019-02-11 at 4 27 10 pm](https://user-images.githubusercontent.com/27962005/52573276-12be4e80-2e1a-11e9-8f94-b773bb8c1470.png)
![screen shot 2019-02-11 at 4 25 38 pm](https://user-images.githubusercontent.com/27962005/52573277-12be4e80-2e1a-11e9-9257-a389b028fed6.png)

**Comment by cmgrote** [comment]
Thanks @alexcontini -- looks good!  Personally I like the middle option: clearly distinguishes the code, and I think will work nicely with the dark background of the sidebar.

Otherwise I'd pick the last option (grey outline) over the first (black outline).

@mandy-chessell any preferences?

**Comment by cmgrote** [comment]
Actually, what happens to single-back-ticked words `like this` in those styles?  All the styles might be a bit severe for that -- I'd prefer to stick to what we have in general in GH for that (light grey background, no outline to the box, mono font)

**Comment by alexcontini** [comment]
@cmgrote sounds good! Let me know if this works when you have a chance - 

![screen shot 2019-02-12 at 11 08 34 am](https://user-images.githubusercontent.com/27962005/52627837-94fc5080-2eb6-11e9-99dc-d507f16cbac6.png)

**Comment by cmgrote** [comment]
Yes, looks good to me!

**Comment by cmgrote** [comment]
Hi @alexcontini -- just noticed one other "gotcha" (maybe you've already addressed it, though?)

A number of our code examples may use different ways of annotating variables: `{}`, `<>`, `[]`, etc.  It seems that at least some of these are being interpreted currently rather than displayed (?)

For example, this page:
https://egeria.odpi.org/open-metadata-implementation/adapters/open-connectors/repository-services-connectors/open-metadata-collection-store-connectors/ibm-igc-repository-connector/

Shows:
```
POST http://localhost:8080/open-metadata/admin-services/users//servers//cohorts/cocoCohort
```

Yet the markdown itself has:
```
POST http://localhost:8080/open-metadata/admin-services/users/{{user}}/servers/{{server}}/cohorts/cocoCohort
```

(See: https://github.com/odpi/egeria/tree/master/open-metadata-implementation/adapters/open-connectors/repository-services-connectors/open-metadata-collection-store-connectors/ibm-igc-repository-connector/README.md)

Basically anything that is surrounded by triple-back-ticks should not be interpreted / replaced, as it is meant to represent exact code...  Any chance that's already been fixed by your updates above? =)

**Comment by alexcontini** [comment]
@cmgrote  sounds good! I created this PR here: https://github.com/odpi/egeria/pull/769 for the first 2 issues, and still trying to figure out how to fix the last one (anything that is surrounded by triple-back-ticks should not be interpreted / replaced) and will submit the PR for that one soon!

**Comment by cmgrote** [comment]
@alexcontini thanks -- just merged, and it's looking better:

<img width="931" alt="screenshot 2019-02-19 at 16 04 24" src="https://user-images.githubusercontent.com/5864283/53029109-22641580-3460-11e9-806e-008c220017bb.png">

Two follow-up thoughts:

1. Would be nice if there's a bit more padding around the box (like examples in discussion above) -- at the moment the background box is very immediate around the text.
1. Looks like we're word-wrapping now rather than allowing the box to be scrolled. For normal text wrapping is definitely the way to go, but for code it can be problematic... Is scrolling an option rather than wrapping the text for these code blocks (triple-back-ticked text)?

**Comment by mandy-chessell** [comment]
is this finished?

**Comment by cmgrote** [comment]
> is this finished?

Should be once #1906 is merged...
